Heejung/Lv2/멀쩡한 사각형.js

```diff
@@ -0,0 +1,12 @@
+function solution(w, h) {
+  const answer = w * h;
+  /* 빼야할 사각형 개수
+    최대공약수 존재x -> w + h - 1
+    최대공약수 존재o -> w + h - (w, h의 최대공약수) */
+  return answer - (w + h - gcd(w, h));

```

Comment:
```suggestion
  /* 빼야할 사각형 개수
    최대공약수 존재x -> w + h - 1
    최대공약수 존재o -> w + h - (w, h의 최대공약수) */
  return w * h - (w + h - gcd(w, h));
```
이 경우에서는 개인적으로 answer를 선언해주는것보다는 return에 바로 적어주는게 좋아보여요.